blivet/devicefactory.py

```diff
@@ -332,7 +332,11 @@ def __init__(self, storage, **kwargs):
             setattr(self, setting, value)
 
         self.fstype = None  # not included in default_settings b/c of special handling below
-        self.min_luks_entropy = kwargs.get("min_luks_entropy") or luks_data.min_entropy
+        self.min_luks_entropy = kwargs.get("min_luks_entropy", None)
```

Comment:
`None` is returned by default by `.get()` if the key doesn't exist.

Reply:
Fixed.